pkg/appgw/health_probes.go

```diff
@@ -148,7 +148,11 @@ func (c *appGwConfigBuilder) generateHealthProbe(backendID backendIdentifier) *n
 			probe.Timeout = to.Int32Ptr(k8sProbeForServiceContainer.TimeoutSeconds)
 		}
 		if k8sProbeForServiceContainer.FailureThreshold != 0 {
-			probe.UnhealthyThreshold = to.Int32Ptr(k8sProbeForServiceContainer.FailureThreshold)
+			if k8sProbeForServiceContainer.FailureThreshold > 20 {
```

Comment:
could you help add a comment why is 20 specifically?

Reply:
Comment added. If you agree with the statement please resolve the converstation.

Reply:
Thanks!